packages/synapse-react-client/test/containers/QueryWrapper.test.tsx

```diff
@@ -64,191 +77,354 @@ const initialQueryRequest: QueryBundleRequest = {
   },
 }
 
-describe('basic functionality', () => {
-  it('renders without crashing', async () => {
-    renderComponent({ initQueryRequest: initialQueryRequest })
-    await screen.findByText(renderedTextConfirmation)
-  })
+describe('QueryWrapper', () => {
```

Comment:
Wrapped in a describe to easily run all at once in IDE

Reply:
The existing tests are essentially unchanged, I would recommend just reviewing the new tests.